### ISSUE TITLE
BUG: ROW Usage fee NOT getting added and Extension fees are getting duplicated with Parking Meter (Property & Street information) Parameters

### ISSUE THREAD

**Issue by Nadin-Nader** [cityofaustin/atd-data-tech#8000]
When Parking Meters (Property & Street information) is added then row-usage fee not getting added and extension fee getting duplicated on Extension Granted attempt in RW-TURP

Stakeholders: Kim

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Started to work on this issue. Analysis is in progress.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Requested a few TEST Examples to replicate and fix the issue.

**Comment by TracyLinder** [comment]
I asked Kim for example RSNs and asked what work-around they're currently using

**Comment by TracyLinder** [comment]
**Response from Kim**
<img width="890" alt="Screen Shot 2022-01-27 at 4 12 56 PM" src="https://user-images.githubusercontent.com/49930868/151452405-f6080c80-abc3-4b08-a61a-caf35d3bc911.png">

**Comment by TracyLinder** [comment]
**I'm going to try to test this myself**

- Portal Application submitted just fine (testing in PROD) - 2022-012603 RW
- The system is inserting the correct ROW usage fees (I haven't verified that the amounts are correct)
- The system did not bill the fees (not sure if this is a bug)
- I'll ask Kim to confirm.

**Comment by TracyLinder** [comment]
**To Kim:**

 I just tested this in PROD 2022-012603 RW. Are you able to confirm if the fee amounts are correct? Here are screenshots of the prop & street info tab & the fees the system added when i saved extension granted for 5 days in tier 1. (I had to delete the fees so i could test a $0 payment.) 
![TURP prop info tab](https://user-images.githubusercontent.com/49930868/151458292-edcf309d-724f-4644-b9fd-eb34143823f6.jpeg)
![TURP Usage Fees](https://user-images.githubusercontent.com/49930868/151458302-e4669edf-65db-43c8-9496-fa2a560cdac5.jpeg)
